packages/google_maps_flutter/lib/src/google_map.dart

```diff
@@ -91,6 +92,9 @@ class GoogleMap extends StatefulWidget {
   /// True if the map view should respond to zoom gestures.
   final bool zoomGesturesEnabled;
 
+  /// True if the map view should be in lite mode. Android only. See https://developers.google.com/maps/documentation/android-sdk/lite#overview_of_lite_mode for more details.
```

Comment:
nits: The `See..` should be at its own paragraph.
```
/// True if the map view should be in lite mode. Android only.
///
/// See https://developers.google.com/maps/documentation/android-sdk/lite#overview_of_lite_mode for more details.
```

Reply:
Done